### PR TITLE
general: Place dialog form labels on top of their fields

### PR DESCRIPTION
**Pull request by mvollmer**
This allows for longer labels.

Pixel changes [here](https://github.com/cockpit-project/pixel-test-reference/compare/8ee2e589cb8448aaf4bd8cf07f22b3cbebc18739..35974bdda540bf95f58042085dbe9137254667b7).

### CODE REVIEW

File: src/components/vm/disks/diskAdd.jsx [hunk 1]
```diff
@@ -207,7 +207,7 @@ const AdditionalOptions = ({ cacheMode, device, idPrefix, onValueChanged, busTyp
     return (
         <ExpandableSection toggleText={ expanded ? _("Hide additional options") : _("Show additional options")}
                            onToggle={() => setExpanded(!expanded)} isExpanded={expanded} className="pf-v5-u-pt-lg">
-            <Form onSubmit={e => e.preventDefault()} isHorizontal>
+            <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/disks/diskAdd.jsx:210]: This added line is not executed by any test.

File: src/components/vm/disks/diskAdd.jsx [hunk 2]
```diff
@@ -584,7 +584,7 @@ export const AddDiskModalBody = ({ disk, idPrefix, isMediaInsertion, vm, vms, su
         );
     } else if (isMediaInsertion) {
         defaultBody = (
-            <Form onSubmit={e => e.preventDefault()} isHorizontal>
+            <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/disks/diskAdd.jsx:587]: This added line is not executed by any test.

File: src/components/vm/disks/diskAdd.jsx [hunk 3]
```diff
@@ -620,7 +620,7 @@ export const AddDiskModalBody = ({ disk, idPrefix, isMediaInsertion, vm, vms, su
     } else {
         defaultBody = (
             <>
-                <Form onSubmit={e => e.preventDefault()} isHorizontal>
+                <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/disks/diskAdd.jsx:623]: This added line is not executed by any test.

File: src/components/vm/nics/nicAdd.jsx [hunk 4]
```diff
@@ -179,7 +179,7 @@ export class AddNIC extends React.Component {
         const { idPrefix, vm } = this.props;
 
         const defaultBody = (
-            <Form onSubmit={e => e.preventDefault()} isHorizontal>
+            <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/nics/nicAdd.jsx:182]: This added line is not executed by any test.

File: src/components/vm/nics/nicEdit.jsx [hunk 5]
```diff
@@ -154,7 +154,7 @@ export class EditNICModal extends React.Component {
         const { idPrefix, vm, network } = this.props;
 
         const defaultBody = (
-            <Form onSubmit={e => e.preventDefault()} isHorizontal>
+            <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/nics/nicEdit.jsx:157]: This added line is not executed by any test.

File: src/components/vm/overview/vsock.jsx [hunk 6]
```diff
@@ -154,7 +154,7 @@ export const VsockModal = ({ vm, vms, vmVsockNormalized, isVsockAttached, idPref
     const vsockUsage = getVsockUsageMessage(vm.name, vm.connectionName, vms, auto, address);
 
     const body = (
-        <Form onSubmit={e => e.preventDefault()} isHorizontal>
+        <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/overview/vsock.jsx:157]: This added line is not executed by any test.

File: src/components/vm/overview/watchdog.jsx [hunk 7]
```diff
@@ -197,7 +197,7 @@ export const WatchdogModal = ({ vm, isWatchdogAttached, isRemovable, idPrefix })
                }>
             {needsShutdown()}
             <WatchdogModalAlert dialogError={dialogError} />
-            <Form onSubmit={e => e.preventDefault()} isHorizontal>
+            <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/overview/watchdog.jsx:200]: This added line is not executed by any test.

File: src/components/vm/snapshots/vmSnapshotsCreateModal.jsx [hunk 8]
```diff
@@ -182,7 +182,7 @@ export class CreateSnapshotModal extends React.Component {
         const validationError = this.onValidate();
 
         const body = (
-            <Form onSubmit={e => e.preventDefault()} isHorizontal>
+            <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/snapshots/vmSnapshotsCreateModal.jsx:185]: This added line is not executed by any test.

File: src/components/vm/vmMigrateDialog.jsx [hunk 9]
```diff
@@ -167,7 +167,7 @@ export const MigrateDialog = ({ vm, connectionName }) => {
     }
 
     const body = (
-        <Form onSubmit={e => e.preventDefault()} isHorizontal>
+        <Form onSubmit={e => e.preventDefault()}>
```
Comment (cockpituous) [inline src/components/vm/vmMigrateDialog.jsx:170]: This added line is not executed by any test.

**Comment by garrett** [comment]
Cool experiment! Thanks!

I looked at the diff @ https://github.com/cockpit-project/pixel-test-reference/compare/8ee2e589cb8448aaf4bd8cf07f22b3cbebc18739..35974bdda540bf95f58042085dbe9137254667b7

I don't know if we want to go all-in on side or top labelling or pick what fits best. We might want to default to top labelling and do side labelling for more complex forms like this?

**Pull request closed without merging**